spec/fixtures/bibliography/rendered.html

```diff
@@ -1,11 +1,11 @@
 <ol class="csl-bibliography">
   <li class="csl-entry" style="margin-bottom: 0.0em"><i>A Declaration of Certayne Principall Articles of Religion: Set out by the Order of Both Archebyshoppes Metropolitans, and the Rest of the Byshoppes, for the Vnitie of Doctrine to Be Taught and Holden of All Parsons, Vicars, and Curates, Aswell in the Testification of Theyr Common Consent in the Sayde Doctrine, to the Stoppyng of the Mouthes of Them That Goo about to Slaunder the Ministers of the Church for Diuersitie of Iudgement: as Necessary for the Instruction of Their People, to Read by the Sayde Parsons, Vicars, and Curates, at Theyr Possession Takyng or First Entrye into Theyr Cures: and Also after That Yerely at Two Seuerall Tymes. That Is to Say, the Sundayes next Folowyng Easter Day and Saint Michaell Tharchaungell, or on Some Other Sunday within One Moneth after Those Feastes, Immediately after the Gospell</i>. 1561. London.</li>
+  <li class="csl-entry" style="margin-bottom: 0.0em">Azevedo, R. de. 1962. <i>A Carta Ou Memória Do Cruzado Inglês R. Para Osberto De Bawdsey Sobre a Conquista De Lisboa Em 1147</i>. 1st ed. Vol. I. Coimbra: Faculdade de Letras da Universidade de Coimbra.</li>
```

Comment:
This is an intentional change from upstream:

https://github.com/citation-style-language/styles/pull/1677